### PR TITLE
✨[InsurAds] amp-ad InsurAds implementation

### PR DESCRIPTION
**Pull request by JoseClimaco**
Add amp-ad implementation for InsurAds

### CODE REVIEW

**Comment by JoseClimaco** [comment]
cc @powerivq

File: 3p/integration.js [hunk 1]
```diff
@@ -145,6 +145,7 @@ import {improvedigital} from '#ads/vendors/improvedigital';
 import {inmobi} from '#ads/vendors/inmobi';
 import {innity} from '#ads/vendors/innity';
 import {insticator} from '#ads/vendors/insticator';
+import {insurads} from '#ads/vendors/insurads';
```
Comment (powerivq) [inline 3p/integration.js:148]: No need to change this file. It was for backwards compatibility. Please revert them.

**Review by powerivq — CHANGES_REQUESTED** [review]
Otherwises looks good

**Review by powerivq — APPROVED** [review]

File: 3p/integration.js (hunk 1 above)
Comment (JoseClimaco) [reply 3p/integration.js]: Done 👍

**Pull request merged**